runtime-kernel/broadcom-wl/autobuild/build

```
@@ -0,0 +1,14 @@
+local dest="${PKGDIR}/usr/src/broadcom-wl-${PKGVER}"
+mkdir -vp "${dest}"
+cp -va src Makefile dkms.conf "${dest}"
+install -Dm0644 lib/wlc_hybrid.o_shipped \
+     "${PKGDIR}/usr/lib/broadcom-wl/wlc_hybrid.o_shipped"
+
+install -Dm644 -t "${dest}/patches" \
+    "$SRCDIR"/autobuild/patches/*
+
+local ldir="${PKGDIR}/usr/share/doc/broadcom-wl"
+mkdir -vp "${ldir}"
+install -Dm0644 lib/LICENSE.txt "${ldir}/LICENSE.shipped"
+sed -n -e '/Copyright/,/SOFTWARE\./{s/^ \* //;p}' \
```

Please make this a source patch. `sed` is unreliable and fails silently.

---

Removed
Short license text stays in the source files. Full license text goes into `/usr/share/doc/broadcom-wl/LICENSE`.